tf/env/local/secrets-minio.tf

```diff
@@ -0,0 +1,17 @@
+resource "random_password" "minio-password" {
+  length = 32
+}
+
+resource "kubernetes_secret" "minio-credentials" {
+  metadata {
+    name = "minio-credentials"
+  }
+
+  binary_data = {
+    "rootPassword" = base64encode(random_password.minio-password.result)
```

Comment:
Considering this is local-only, we could also hard code a secret key. Any opinions?

Reply:
no strong opinion - I think both would be fine